simulationsview.cpp

```diff
@@ -164,12 +155,14 @@ void SimulationsView::saveImageNamed(QString fileName){
     svImage.save(fileName);
 }
 
-void SimulationsView::showContextMenu(QPoint pos){
-    // Handle global position
+void SimulationsView::showContextMenu(const QPoint &pos)
+{
+
+// Handle global position
     QPoint globalPos = mapToGlobal(pos);
     // Create menu and insert some actions
     QMenu myMenu;
-    myMenu.addAction("Save as image", this, &SimulationsView::saveImage); // connects to QAction::triggered(bool checked = false)
+    myMenu.addAction("Save as image",  this, SLOT(saveImage()));
```

Comment:
```suggestion
    myMenu.addAction("Save as image", this, &SimulationsView::saveImage); // connects to QAction::triggered(bool checked = false)
```
regression from main